libcrux-iot/ml-dsa/src/encoding/t0.rs

```diff
@@ -12,7 +14,7 @@ const OUTPUT_BYTES_PER_SIMD_UNIT: usize = 13;
 #[inline(always)]
 pub(crate) fn serialize<SIMDUnit: Operations>(
     re: &PolynomialRingElement<SIMDUnit>,
-    serialized: &mut [u8], // RING_ELEMENT_OF_T0S_SIZE
+    serialized: &mut [U8], // RING_ELEMENT_OF_T0S_SIZE
```

Comment:
I don't know what this ring element is - but if the bytes are secret, maybe the ring element also should be? Not sure.